### DISCUSSION TITLE
Code coverage for the .NET application using robot framework automation test cases

### DISCUSSION THREAD

**Discussion by RamyaKT** [coverlet-coverage/coverlet#1638] [Q&A] [answered]
Hi Team,

I have the source code in .NET application. but for automation, I have written test cases in robot framework (on top of python). From the QA perspective, I wanted to get the code coverage through running automation test cases. 

is there any way to achieve this using coverlet?

I am not finding the relevant documents / details regarding cross language platform. could anyone please help me out for this.
If anyone having the examples on connecting 2 different platforms would also be very helpful.

**Comment by daveMueller** [comment]
Coverlet is a bit limited related to test execution started by another executeable. But you can give `coverlet.console` at try. Here is a short description how to use it for end-to-end tests (https://github.com/coverlet-coverage/coverlet/blob/master/Documentation/GlobalTool.md#code-coverage-for-integration-tests-and-end-to-end-tests)

**Reply by RamyaKT** [reply]
but I do not have any test folder to instrument. can I leave it as it is for that section

**Reply by daveMueller** [reply, marked as answer]
Never worked with the robot framework but I would guess that somewhere your C# source code will be compiled to an assembly. The directory of this assembly must be specified.
